scripts/qa_crowbarsetup.sh

```diff
@@ -2974,6 +2974,10 @@ function custom_configuration()
                 local cinder_volume
                 # fetch one of the compute nodes as cinder_volume
                 cinder_volume=`printf "%s\n" $unclustered_nodes | tail -n 1`
+                # make sure we do not use SP1 node for cinder
+                if [ -n "$deployceph" -a iscloudver 7plus ] ; then
```

Comment:
> If deployceph and is a SOC7, cinder_volume is a node with target=suse-12.1

@aplanas Seems you were looking at an old version of the code ( @jsuchome 's master branch is outdated). The current master picks an SP2 node when running on cloud 7